lib/message_bus/rack/middleware.rb

```diff
@@ -139,6 +139,20 @@ def handle_request(env)
 
     backlog = client.backlog
 
+    if @bus.on_middleware_attributes
+      @bus.on_middleware_attributes.call(
+        messagebus_seq: client.seq,
```

Comment:
I think this is my biggest issue with this change, would prefer if we just pass env along and a fatter object vs all these options.